### DISCUSSION TITLE
Listener timeout even though it did not exceed 10s

### DISCUSSION THREAD

**Discussion by jonas-martinez** [lenra-io/lenra_cli#105] [Q&A]
### The issue

https://github.com/jonas-martinez/achiever

For a bit of context, the application "achiever" is made to fetch your steam games and track your statistics and achievements. This makes the app store a lot of information inside datastores. For example, my Steam account has more than 250 games which means that it will call the `POST /app/datastores/games/data` more than 250 times. The total time to execute these requests is approximately of 2 to 3 seconds (I counted in my head, anyway it's just to say that it does not take more than 10seconds).

These requests are made when clicking on a button after giving your SteamID. I'm wondering what could be the reason for the timeout to stop before the 10 seconds mark.

### How to reproduce

You can reproduce this by using the following branch of the achiever repository : https://github.com/jonas-martinez/achiever/tree/lenra-bug-timeout

When the app asks for your steamID, you can use mine `76561197973393048`

**Comment by taorepoara** [comment]
Resolved by defining the of-watchdog timeouts with the CLI
